src/main/java/seedu/moneygowhere/model/path/FilePath.java

```diff
@@ -0,0 +1,48 @@
+package seedu.moneygowhere.model.path;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.moneygowhere.commons.util.AppUtil.checkArgument;
+
+import seedu.moneygowhere.commons.util.FileUtil;
+
+/**
+ * Represents a file path.
+ */
+public class FilePath {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "File path should not contain any of the following characters: < > ? [ ] : | * .  \n"
+                    + "It should not not contain more than 218 characters and it should be an existing csv file.";
```

Comment:
**The file** should not contain more than 218 characters

Reply:
Why? The file can contain more than 218 characters, the path can’t

Reply:
Grammar issue, can probably replace `it` to  `and not contain more than 218 characters`